Makefile

```diff
@@ -1,7 +1,9 @@
 #!/usr/bin/make -f
 
 SHELL=bash
-GOBIN ?= $(shell go env GOPATH)/bin
+# GOBIN is used only to install neo-go and allows to override
+# the location of written binary.
+export GOBIN ?= $(shell pwd)/bin
```

Comment:
does `CURDIR` make the same?

Reply:
It does, but it is a GNU extension, I am not sure where if this can bite us.